swagger.yml

```
@@ -1180,15 +1180,21 @@ definitions:
         items:
           $ref: '#/definitions/versicherungsrisiken'
       versicherer:
-        description: ID eines Produktanbieters.
+        description: Veraltet! Bitte versichererIds verwenden.
         type: string
+      versichererIds:
+        description: IDs der Versicherer. Pflichtangabe, falls versicherer Property nicht gesetzt ist.
```

Ich bin dafür hier gleich den Hinweis aufzunehmen dass die ID von Europace vorgegeben wird.
```suggestion
        description: IDs der Versicherer. Gültige Ausprägungen erhalten Sie von ihrem Ansprechpartner bei Europace. Pflichtangabe, falls versicherer Property nicht gesetzt ist.
```